README.md

```diff
@@ -196,59 +221,31 @@ If you want to use a custom database, you will need to create a Sourmash sketch
 If you have a single FASTA file with _one genome_ per record:
 
 ```bash
-sourmash sketch dna -f -p k=31,scaled=1000,abund --singleton <your multi-FASTA file> -o training_database.sig.zip
+yacht sketch ref --infile <path to your multi-FASTA file> --kmer 31 --scaled 1000 --outfile training_database.sig.zip
 ```
 
 If you have a directory of FASTA files, one per genome:
 
 ```bash
-## Method 1 (suggested)
-# put all full paths of FASTA/FASTQ file into a file, one path per line
-find <path of foler containg FASTA/FASTQ files> > dataset.csv
-sourmash sketch fromfile dataset.csv -p dna,k=31,scaled=1000,abund -o ../training_database.sig.zip
-# cd back to YACHT
-
-## Method 2
-# cd into the relevant directory
-sourmash sketch dna -f -p k=31,scaled=1000,abund *.fasta -o ../training_database.sig.zip
-# cd back to YACHT
+yacht sketch ref --infile <path of foler containg FASTA/FASTQ files> --kmer 31 --scaled 1000 --outfile training_database.sig.zip
 ```
 
 </br>
 
 
-### Creating sketches of your sample
+### Creating sketches of your sample (yacht sketch sample)
 
 Creating a sketch of your sample metagenome is an essential step in the YACHT workflow. This process involves using the same k-mer size and scale factor that were used for the reference database. You can use the following commands to implement this step:
 
 ```bash
 # For a single-end FASTA/Q file
-sourmash sketch dna -f -p k=31,scaled=1000,abund -o sample.sig.zip <input FASTA/Q file>
+yacht sample --infile <input FASTA/Q file> --kmer 31 --scaled 1000 --outfile sample.sig.zip
 
-# For pair-end FASTA/Q files, you need to combine them into a single file first
-cat <FASTA/Q file 1> <FASTA/Q file 2> > combine.fastq (or combine.fasta)
-sourmash sketch dna -f -p k=31,scaled=1000,abund -o sample.sig.zip combine.fastq (or combine.fasta)
+# For pair-end FASTA/Q files, you need to separately specify two FASTA/Q files
+yacht sample --infile <FASTA/Q file 1> <FASTA/Q file 2> --kmer 31 --scaled 1000 --outfile sample.sig.zip
 ```
 
-#### Parameters
-
-Sourmash database offers three available k values (21, 31, and 51), allowing you to select the one that best suits your particular analytical needs. The scale factor serves as an indicator of data compression, and if your dataset is small, you might consider using a smaller value (corresponding to a higher portion of genomes retained in the sketch).
-
-| Parameter         | Explanation                                                  |
-| ----------------- | ------------------------------------------------------------ |
-| k=31              | the length of k-mer                                          |
-| scaled=1000       | fraction of k-mers to be kept in the sketch                  |
-| -o sample.sig.zip | arbitrary output name for sketch files (must be in zip format) |
-
-#### Output
```

Comment:
Was there a particular reason to delete this part? Seems still relevant since those files will exist after running `yacht sketch ...`

Reply:
The reason why I deleted this part is because we now mainly use `yacht sketch ref/sample` instead of `sourmash`. I have already illustrated the parameters [here](https://github.com/KoslickiLab/YACHT/blob/474208ff6845d9a634737e180cb4681773af0397/README.md?plain=1#L160-L180) for these new commands.